boards/yunjia-nrf51822/Makefile.dep

```diff
@@ -0,0 +1,3 @@
+ifneq (,$(filter ng_ble,$(USEMODULE)))
+  USEMODULE += ng_ble
```

Comment:
Typo? Currently you add ng_ble into the USEMODULE set, if it already is in the set.
Did you want to add it if it is absent? Then you can just add it without a guard, because duplicates are unified in a later step.